lib/git/config.rb

```diff
@@ -10,7 +10,7 @@ def initialize
     end
 
     def binary_path
-      @binary_path || 'git'
+      @binary_path || ENV['GIT_BIN'] || 'git'
```

Comment:
How about `GIT_PATH` instead? Or `GIT_BINARY`? (I would prefer the first option, but interested in hearing your opinions also.)

Reply:
In my opinion, pointing to the actual binary is the best way to give users the flexibility that they need. Though I can see the confusion since the method name is `binary_path`. I could rename the function if that makes it more clear since it is not returning the path of the binary but the binary itself.

What do you think @perlun?

Reply:
@afiune Thanks. I was just thinking about changing the environment variable name, sorry for not being so clear here. 👍

Reply:
Looked at your change now. I think `GIT_PATH` would be the clearer way. Sorry for being slightly picky, but please change the env variable to that and I'll be happy to get this merged ASAP.

Reply:
Sounds good. I’ll make the change, update the tests and documentation. 💯